support/xspec-dev/readme.md

```diff
@@ -0,0 +1,96 @@
+
+The HTML XSLT was developed to mitigate configuration problems using the project-standard presentation stylesheet (as noted below). It is designed to be functional but lightweight, robust and cleanly extensible for a range of use cases or deployments.
+
+*Limitation*: At present, the presentation stylesheet meets these goals with respect to XSpec testing XSLT - not (yet) testing XQuery, Schematron or other runtimes to which XSpec might be applied.
+
+`xspec-batch-report.xpl` - provides the same set of operations, except instead of a single XSpec (file instance), it can accept any number of XSpec files designated as inputs (either wired in or via XProc port bindings). Results from these tests are *aggregated* into a single report.
+
+This pipeline also demonstrates how reports can be further processed to provide other outputs such as XML summaries (for batches) and simple plain-text summaries.
+
+Additionally, another XProc is included for future diagnostics:
+
+`xspec-single-xspec-repo-report.xpl` - calls the XSpec project XProc (pipeline), delivering an HTML file using the 'regular' XSLT in the XSpec distribution. *Note*: YMMV especially as it regards CSS setup in HTML file results.
+
+This pipeline is essentially a minimal, functionally standalone wrapper around XSpec-native capabilities, made to expose the 'pain points' in using the current resources (which were originally authored and tested over ten years ago). For good results using XProc 1.0 under XML Calabash (itself no longer cutting-edge technology), certain modifications must be made in the XSpec repository libraries:
+
+* Serialization needs to use latest (HTML5) serialization options to deal with character handling, rather than the current 'double reverse' character escaping of markup characters into the Unicode PUA - thus effectively requiring a character mapping back, which is not well supported under XProc 1.0.
+* CSS handling/deployment needs to be improved or exposed such that generated HTML files have their CSS available.
+
+## How to batch XSpec - a 'mini' demo
+
+A demonstration of how to use the **batch report** XProc as a step in a calling XProc is provided. The advantage of such a 'wrapper' XProc is that it can encapsulate logic for an even simpler interface or runtime for the specific use case (whether running from the command line, under CI/CD, in an IDE etc). Both the batch (the XSpecs to be run) and what to do with the test results can thus be hard-wired, or provided with customized interface controls.
+
+The ['test batch' XProc](testing/xspec-test-batch.xpl) in this folder can be copied anywhere and adjusted per case, restoring its import link, pointing it to local XSpec file sources, and setting up ports or `p:store` (file save) options as wanted. It then runs in place to execute, as a set, all the XSpecs it points to.
```

Comment:
Upon second review, it may help if you given more hand-holding here, as the first part of my review summarizing my trouble appears to be this but I am not sure reading the XSpec issue and these docs exactly what the relative import should be.

Reply:
Fixing that bug would undoubtedly help a great deal.

This is why we do code review, thank you.

(Too bad we have no XSpec for XProc, ha. How to smoke-test an XProc run is an interesting question.)